selenium/exercise-1.rb

```diff
@@ -0,0 +1,9 @@
+require 'selenium-webdriver'
+
+Selenium::WebDriver::Chrome::Service.driver_path = "C:/Users/anup.desai/Desktop/infuse/Practice-ANUPDESAI/selenium/geckodriver.exe"
```

Comment:
To open the Firefox driver you need to set the Firefox driver path, you have incorrectly set the chrome driver path to point to the firefox driver.

Reply:
Comment applies to all files